src/timesync/timesyncd-bus.c

```diff
@@ -63,6 +126,22 @@ static const sd_bus_vtable manager_vtable[] = {
         SD_BUS_PROPERTY("RootDistanceMaxUSec", "t", bus_property_get_usec, offsetof(Manager, max_root_distance_usec), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("PollIntervalMinUSec", "t", bus_property_get_usec, offsetof(Manager, poll_interval_min_usec), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("PollIntervalMaxUSec", "t", bus_property_get_usec, offsetof(Manager, poll_interval_max_usec), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("PollIntervalUSec", "t", bus_property_get_usec, offsetof(Manager, poll_interval_usec), SD_BUS_VTABLE_PROPERTY_CONST),
+        /* NTP response */
+        SD_BUS_PROPERTY("Leap", "u", property_get_ntp_field, offsetof(Manager, ntpmsg), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("Version", "u", property_get_ntp_field, offsetof(Manager, ntpmsg), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("Mode", "u", property_get_ntp_field, offsetof(Manager, ntpmsg), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("Stratum", "y", NULL, offsetof(Manager, ntpmsg.stratum), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("Precision", "i", NULL, offsetof(Manager, ntpmsg.precision), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("Reference", "s", property_get_ntp_reference, offsetof(Manager, ntpmsg), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("OriginateTimestamp", "t", bus_property_get_usec, offsetof(Manager, origin_usec), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("ReceiveTimestamp", "t", bus_property_get_usec, offsetof(Manager, receive_usec), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("TransmitTimestamp", "t", bus_property_get_usec, offsetof(Manager, transmit_usec), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("DestinationTimestamp", "t", bus_property_get_usec, offsetof(Manager, dest_usec), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("RootDistanceUSec", "t", bus_property_get_usec, offsetof(Manager, root_distance_usec), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("Spike", "b", bus_property_get_bool, offsetof(Manager, spike), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("PacketCount", "t", NULL, offsetof(Manager, packet_count), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("JitterUSec", "t", property_get_jitter, offsetof(Manager, samples_jitter), SD_BUS_VTABLE_PROPERTY_CONST),
```

Comment:
hmm, so i wonder if it wouldn't be better to expose this as a single structure, i.e. as one property with type "(uuuyistttttbtt)". I mean, it belongs logically together, right, and we got it as one conceptual unit (i.e. as one datagram), hence it should probably be exposed as one conceptual unit (i.e. as one property), too?

Reply:
OK. Done.